cibyl/sources/elasticsearch/api.py

```diff
@@ -154,6 +160,10 @@ def get_builds(self: object, **kwargs: Argument):
                         build['_source']['build_result'] not in build_statuses:
                     continue
 
+                if 'build_number' in kwargs and \
```

Comment:
This is also unnecessary

Reply:
Since `--build-number` has been deleted, yes, we can remove it. I didn't realize to do this change from the moment it wasn't deleted of the `Job` class. Done.